.github/workflows/e2e-tests.yml

```diff
@@ -184,6 +184,46 @@ jobs:
           - script: |
               cd e2e/env-int-key
               env 1=1 ../../bin/phpstan analyse test.php
+          - script: |
+              cd e2e/trait-caching
+              ../../bin/phpstan clear-result-cache
+              ../../bin/phpstan analyze --no-progress --level 8 --error-format raw data/
+              ../../bin/phpstan clear-result-cache
```

Comment:
Doesn't this second clear-result-cache defeat the purpose of the test?

Reply:
You are right, it is the same behavior as before, but my comment still stands.. wasn't it broken from the beginning?

Reply:
you might be right. I don't know the specifics of the bug which this test is covering.

if you are right, it would mean the test-as it was before this PR did not reproduce the actual bug.